Command/LoadFixtureCommand.php

```diff
@@ -135,7 +135,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ($dirOrFile) {
             $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
         } else {
-            $paths = array();
+            /** @var $kernel \Symfony\Component\HttpKernel\KernelInterface */
+            $kernel = $this->getApplication()->getKernel();
+            $paths = array($kernel->getRootDir().'/DataFixtures/PHPCR');
             foreach ($this->getApplication()->getKernel()->getBundles() as $bundle) {
```

Comment:
please change this to $kernel->getBundles(), no need to get application and kernel twice